apps/omg_child_chain/lib/omg_child_chain/application.ex

```diff
@@ -34,14 +35,31 @@ defmodule OMG.ChildChain.Application do
   end
 
   def start_phase(:attach_telemetry, :normal, _phase_args) do
+    config = %{
+      # DD reserved tag keys -- cannot use: host, device, service, source
+      # TODO(jacob) cleanup
+      tags: [
```

Comment:
whats the purpose of this? tags are already attached to global statix configuration

Reply:
yep, an expirement to check something. deleting.